arcgis-ios-sdk-samples/Layers/Export tiles/ExportTilesViewController.swift

```diff
@@ -118,39 +118,39 @@ class ExportTilesViewController: UIViewController {
         self.deleteAllTpks()
         
         //initialize the export task
-        self.exportTask = AGSExportTileCacheTask(URL: self.tiledLayer.URL!)
-        self.exportTask.exportTileCacheParametersWithAreaOfInterest(self.frameToExtent(), minScale: self.mapView.mapScale, maxScale: self.tiledLayer.maxScale) { [weak self] (params: AGSExportTileCacheParameters?, error: NSError?) in
+        self.exportTask = AGSExportTileCacheTask(url: self.tiledLayer.url!)
+        self.exportTask.exportTileCacheParameters(withAreaOfInterest: self.frameToExtent(), minScale: self.mapView.mapScale, maxScale: self.tiledLayer.maxScale) { [weak self] (params: AGSExportTileCacheParameters?, error: Error?) in
             if let error = error {
-                SVProgressHUD.showErrorWithStatus(error.localizedDescription, maskType: .Gradient)
+                SVProgressHUD.showError(withStatus: error.localizedDescription, maskType: .gradient)
             }
             else {
                 self?.exportTilesUsingParameters(params!)
             }
         }
     }
     
-    private func exportTilesUsingParameters(params: AGSExportTileCacheParameters) {
+    fileprivate func exportTilesUsingParameters(_ params: AGSExportTileCacheParameters) {
         //destination path for the tpk, including name
-        let path = NSSearchPathForDirectoriesInDomains(.DocumentDirectory, .UserDomainMask, true)[0]
+        let path = NSSearchPathForDirectoriesInDomains(.documentDirectory, .userDomainMask, true)[0]
         let destinationPath = "\(path)/myTileCache.tpk"
         
         //get the job
-        self.job = self.exportTask.exportTileCacheJobWithParameters(params, downloadFileURL: NSURL(string: destinationPath)!)
+        self.job = self.exportTask.exportTileCacheJob(with: params, downloadFileURL: URL(string: destinationPath)!)
         //run the job
-        self.job.startWithStatusHandler({ (status: AGSJobStatus) -> Void in
+        self.job.start(statusHandler: { (status: AGSJobStatus) -> Void in
```

Comment:
Two completion blocks in the same call. The last one already has the right syntax